.circleci/config.yml

```diff
@@ -64,6 +64,7 @@ jobs:
               echo "./.ci/pitest.sh pitest-common"  >> commands.txt
               echo "./.ci/pitest.sh pitest-imports" >> commands.txt
               echo "./.ci/pitest.sh pitest-ant"     >> commands.txt
+              echo "./.ci/pitest.sh pitest-meta"    >> commands.txt
```

Comment:
execution is ignored .... as we has limit for 4 containers
please activate https://github.com/checkstyle/checkstyle/blob/master/.circleci/config.yml#L261
and put in `pitest7`

Reply:
as we ignore it and we know it is not finishing ...
please put in pitest7 but keep it commented out

Reply:
I think we should remove this line altogether, as we dont have `pitest-meta` in `pom.xml`

Reply:
Done